### PR TITLE
Hunting queries windows event batch3

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete
   
Change(s):
Add Windows Event Data to the following Hunting Queries:

Hunting Queries/SecurityEvent/powershell_downloads.yaml
Hunting Queries/SecurityEvent/RemoteLoginPerformedwithWMI.yaml
Hunting Queries/SecurityEvent/RemoteScheduledTaskCreationUpdateviaSchtasks.yaml
Hunting Queries/SecurityEvent/Suspicious_enumeration_using_adfind.yaml
Hunting Queries/SecurityEvent/User Logons By Logon Type.yaml
Hunting Queries/SecurityEvent/UserAdd_RemToGroupByUnauthorizedUser.yaml
Hunting Queries/SecurityEvent/UserCreatedByUnauthorizedUser.yaml

Reason for Change(s):
Part of adding Windows Event data source project

Version Updated:
No
Testing Completed:
Yes
Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Hunting Queries/SecurityEvent/powershell_downloads.yaml [hunk 1]
```diff
@@ -6,18 +6,33 @@ requiredDataConnectors:
...
+  | extend CommandLine = tostring(EventData.CommandLine) 
+  | extend ParentProcessName = tostring(EventData.ParentProcessName)
+  | project  TimeGenerated, ComputerName=Computer,AccountName=SubjectUserName,AccountDomain=SubjectDomainName,
+    FileName=tostring(split(NewProcessName, '\\')[-1]),
+  ProcessCommandLine = CommandLine, 
+  InitiatingProcessFileName=ParentProcessName,InitiatingProcessCommandLine="",InitiatingProcessParentFileName="")
+  );
   processEvents};
   ProcessCreationEvents
   | where FileName in~ ("powershell.exe", "powershell_ise.exe","pwsh.exe")
```
Comment (oshezaf) [inline Hunting Queries/SecurityEvent/powershell_downloads.yaml:38]: Add those using "EventData has" above.

**Review by oshezaf — CHANGES_REQUESTED** [review]

**Comment by vmanojreddy** [comment]
@aprakash13  Please review hunting queries. thanks

**Comment by vmanojreddy** [comment]
@yaelrbergman please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@yaelrbergman please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@oshezaf Can you please check and approve the changes. thanks!!!

**Comment by v-spadarthi** [comment]
@oshezaf: Could you please check and approve the changes. thanks!!!

**Comment by v-mchatla** [comment]
Hi @oshezaf
Can we close this and reopen whenever required as you have labelled it as differed.
Thanks

**Comment by oshezaf** [comment]
Done

**Pull request closed without merging**